clients/admin-ui/next.config.js

```diff
@@ -26,11 +26,11 @@ const nextConfig = {
     return [
       {
         source: `/api/v1/:path`,
-        destination: "http://0.0.0.0:8080/api/v1/:path/",
+        destination: "http://fidesctl:8080/api/v1/:path/",
```

Comment:
@allisonking I had to change this to get it to work from within the docker container, is it possible to inject this at run-time? I don't like that this forces development through docker but I also don't want to prevent running the application via docker

Reply:
hmmm we should be able to use .env files to work around this. perhaps the docker container can add a line to `.env.development` (and possibly also `.env.production`?) that sets some variable like `NEXT_PUBLIC_FIDESCTL_API_SERVER=http://fidesctl:8080` and then in this line doing 

```js
destination: `${NEXT_PUBLIC_FIDESCTL_API_SERVER}/api/v1/:path/`
```

Reply:
updated, thank you for the sugggestion!